lib/utils/fileutils.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+// place for small file utilities. larger parts should move into their own module.
+const fs = require('fs-extra');
```

Comment:
I don't thin this whole file is necessary. You can do the tests in functions that need it, they are small.
Also, this is not reused code yet, and I would only put it in new files if it is reused, as a refactoring step.

Reply:
Wouldn't we see usage of checking if file exists and is not empty at multiple locations. Also keeping it in a separate utils helps run unit tests on them easily

Reply:
Currently we don't reuse this, so no need to bloat the SDK with new files. We'll refactor if/when needed.

Reply:
Removed the util file and moved it into storage.js